pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -722,15 +752,25 @@ public static Map<String, List<KitRequestShipping>> getAllKitRequestsByRealm(@No
         return kitRequests;
     }
 
+    public static Map<String, List<KitRequestShipping>> getAllKitRequestsByRealmForES(@NonNull String realm, boolean getAll) {
+        logger.info("Collecting kit information for ES export");
+        //this method will include deactivated kits as well even if it is reactivated so that elastic has the deactivated record
+        Map<String, List<KitRequestShipping>> kitRequests = getKitRequestsByKitTypeId(realm, null, null, getAll, SQL_SELECT_KIT_REQUEST_FOR_ES);
+        return kitRequests;
+    }
+
     private static PreparedStatement getSelectKitReqPreparedStatement(@NonNull Connection conn, String target, @NonNull String realm,
-                                                                      Integer kitTypeId, boolean getAll) throws SQLException {
+                                                                      Integer kitTypeId, boolean getAll, String selectQuery) throws SQLException {
         PreparedStatement stmt = null;
+        if (selectQuery == null) {
+            selectQuery = SQL_SELECT_KIT_REQUEST;
```

Comment:
By default use existing SQL which excludes de-activated records if the kit-request was re-activated so that ALL existing functionality excluding ES export is not impacted